extensions/geode-modules-tomcat8/src/distributedTest/java/org/apache/geode/modules/session/Tomcat8SessionsClientServerDUnitTest.java

```diff
@@ -21,41 +21,29 @@
 
 import javax.security.auth.message.config.AuthConfigFactory;
 
+import org.junit.After;
+import org.junit.Before;
 import org.junit.experimental.categories.Category;
+import org.springframework.util.SocketUtils;
 
 import org.apache.geode.cache.Cache;
 import org.apache.geode.cache.CacheFactory;
+import org.apache.geode.cache.client.ClientCache;
 import org.apache.geode.cache.client.ClientCacheFactory;
 import org.apache.geode.cache.server.CacheServer;
-import org.apache.geode.internal.AvailablePortHelper;
-import org.apache.geode.internal.cache.GemFireCacheImpl;
 import org.apache.geode.modules.session.catalina.ClientServerCacheLifecycleListener;
 import org.apache.geode.modules.session.catalina.DeltaSessionManager;
 import org.apache.geode.modules.session.catalina.Tomcat8DeltaSessionManager;
-import org.apache.geode.test.dunit.Host;
+import org.apache.geode.test.dunit.VM;
 import org.apache.geode.test.junit.categories.SessionTest;
 
-@Category({SessionTest.class})
+@Category(SessionTest.class)
 public class Tomcat8SessionsClientServerDUnitTest extends TestSessionsTomcat8Base {
+  private ClientCache clientCache;
 
-  // Set up the session manager we need
-  @Override
-  public void postSetUp() throws Exception {
-    setupServer();
-  }
-
-  @Override
```

Comment:
I think generally speaking we try to put the setup and teardown methods next to eachother.  I've noticed that seems to be the organizational convention to keep these together in test files.

Reply:
Yeah, I re-arranged the methods as part of the second commit (9136cdf1bbc9d2043e08860a7b42d5debbacdf4f).